### PR TITLE
update readme and devDependencies, lint, fix broken tests, add circle ci config and package-lock.json, add prettier

### PR DESCRIPTION
**Pull request by mjhea0**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,21 +36,23 @@
     "vuex": "^3.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.0",
     "@babel/preset-env": "^7.3.1",
     "@vue/cli-plugin-babel": "^3.3.0",
     "@vue/cli-service": "^3.3.0",
     "@vue/test-utils": "^1.0.0-beta.29",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (paltman) [inline package.json]: @mjhea0 why drop back to the bridge release rather than using 7.1+?

File: src/components/widgets/index.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
-import HomerWidget from './homer/HomerWidget.vue';
+import HomerWidget from "./homer/HomerWidget.vue";
```
Comment (paltman) [inline src/components/widgets/index.js]: @mjhea0 is it prettier making the decision to move from single-quotes to double-quotes?  this isn't something we've completely had a decision on like we've done for Python (double-quotes on Python is Eldarion/Pinax settled style), but i've been using single-quotes on all js.  I'm don't really have strong feelings either way and if the defaults here in the lint/prettier are enforcing double-quotes, I'm fine to roll with it.  If it's just as easy to use single-quotes, that does seem more common in js.  I don't know.  Again, no strong feelings but do want us to settle it here and make everything else consistent one way or another.

File: src/components/widgets/index.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const widgets = [
   BookSelectWidget,
   InterlinearWidget,
   PassageWidget,
-  SelectablePassageWidget,
+  SelectablePassageWidget
```
Comment (paltman) [inline src/components/widgets/index.js]: @mjhea0 I do prefer trailing commas though.  Anyway we can configure that as something that doesn't get cleaned up and in fact enforced?

File: package.json (hunk 1 above)
Comment (mjhea0) [reply package.json]: Not all of the Babel dependencies are on version 7 yet. There's a sub-dependency somewhere that still depends on 6. That's why the bridge is needed. https://github.com/babel/babel-bridge

File: src/components/widgets/index.js (hunk 2 above)
Comment (mjhea0) [reply src/components/widgets/index.js]: Yes, prettier was making this decision. I updated the prettier config to allow the use of single quotes.

File: src/components/widgets/index.js (hunk 3 above)
Comment (mjhea0) [reply src/components/widgets/index.js]: Yes, fixed this.

**Comment by paltman** [comment]
Thanks @mjhea0!

**Pull request merged**